src/software/firmware/srcs/battery.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+    Copyright (C) 2020 Makers For Life
+*/
+/******************************************************************************
+ * @author Makers For Life
+ * @file battery.cpp
+ * @brief Battery related functions
+ *****************************************************************************/
+
+#pragma once
+
+// INCLUDES ===================================================================
+
+// Associated header
+#include "../includes/battery.h"
+
+// Externals
+#include "Arduino.h"
+
+// Internal
+#include "../includes/parameters.h"
+
+// PROGRAM =====================================================================
+
+uint16_t DEFAULT_BATTERY_VOLTAGE = 20;  // 20 volts;
```

Comment:
Maybe for consistency with the rest of the code, separate (and prefix with static const) constants, and use lower case for variables?